rpc_jobs/ops_fabric_chatbot.yml

```diff
@@ -0,0 +1,53 @@
+- job:
+    name: ops-fabric-chatbot-gate
+    description: "lint and unit tests for ops-fabric-chatbot"
+    disabled: false
+    display-name: "ops-fabric-chatbot lint and unit tests"
+    project-type: workflow
+    concurrent: true
+    properties:
+      - github:
+          url: https://github.com/rcbops/ops-fabric-chatbot/
+      - build-discarder:
+          num-to-keep: 30
+    parameters:
+      - rpc_gating_params
+      - instance_params:
+          IMAGE: "Ubuntu 16.04 LTS (Xenial Xerus) (PVHVM)"
+          FLAVOR: "performance1-2"
+          REGION: "DFW"
+      - string:
+          name: STAGES
+          default: "Allocate Resources, Connect Slave, Cleanup, Destroy Slave"
+          description: |
+            Pipeline stages to run CSV. Note that this list does not influence execution order.
+            Options:
+              Allocate Resources
+              Connect Slave
+              Pause (use to hold instance for investigation before cleanup)
+              Cleanup
+              Destroy Slave
+    triggers:
+        - github-pull-request:
+            org-list:
+                - rcbops
+            github-hooks: true
+            trigger-phrase: 'recheck'
+            only-trigger-phrase: false
+            auth-id: "github_account_rpc_jenkins_svc"
+            status-context: 'ops-fabric-chatbot-gate'
+    dsl: |
+      library "rpc-gating@${env.RPC_GATING_BRANCH}"
+      common.shared_slave(){
+        pubcloud.runonpubcloud(){
```

Comment:
Did you try running this in a container, to avoid having to spin up a cloud instance?